src/main/java/org/spongepowered/common/event/tracking/phase/packet/InteractionPacketState.java

```diff
@@ -123,6 +123,7 @@ public void unwind(BasicPacketContext phaseContext) {
         final ItemStackSnapshot usedSnapshot = ItemStackUtil.snapshotOf(usedStack);
         final Entity spongePlayer = EntityUtil.fromNative(player);
         try (CauseStackManager.StackFrame frame = Sponge.getCauseStackManager().pushCauseFrame()) {
+            phaseContext.addNotifierAndOwnerToCauseStack();
```

Comment:
It's an interaction packet state.... This should not be done at all since it's a *player packet being processed*...